product-catalog-service/src/main/java/agh/fis/product_catalog/model/Product.java

```diff
@@ -7,31 +7,31 @@
 import javax.persistence.GenerationType;
 import javax.persistence.Column;
 
-import java.sql.Date;
+import java.sql.Date; 
 
 @Entity
 public class Product extends ProductBaseEntity{
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private int id;
 
-    @Column(nullable = false)
+    @Column(nullable = false, columnDefinition="TEXT")
```

Comment:
I think `columnDefinition="TEXT"` is unnecessary, isn't it?

Reply:
It is necessary in order to have string longer than 256, however it still doesnt work